Special purpose language: No linguistic content (zxx)

jesus2099 opened the pull request:
- Like `[Artificial (Other)]` (`qaa`)
- Like `[Multiple languages]` (`mul`)
- Like `[No lyrics]` (same `zxx`)

`No linguistic language` (`zxx`) should be renamed to `[No linguistic language]` (`zxx`) and should thus appear in release language list (select combo) after ~~`[Multiple languages]` (Frequently used)~~ `[Artificial (Other)]`, instead of between `Niuean` and `Nogai`.

I guess it also requires a Database edit of the stored label in languages table.

Maybe I am missing some files.

reosarevok commented:
For what it's worth, the po files should not be changed by hand, and the language renaming, if we decide to do it, should be done by an admin via the site (languages are not hardcoded in code).

jesus2099 commented:
I have created https://tickets.metabrainz.org/browse/MBS-13152 for this, then.
Are there any files that should be kept in this PR or do I cancel it?

reosarevok commented on admin/SetLanguageFrequencies:
```diff
@@ -13,11 +13,11 @@ use Try::Tiny;
 # pseudo language.
 #
 use constant PRESELECTED_LANGUAGES => qw(
-    ara eng fra spa rus zho mul
```
This would set no linguistic content into the top "frequently used" list, which I understand from the ticket you don't think is needed?

jesus2099 replied:
Exacly y'oure righjt!

jesus2099 replied:
Reverted. :)

reosarevok reviewed (commented):
The changes to tests seem fine if we change the name from the DB, but I think everything else is unwanted.

jesus2099 commented:
I have removed `*.po` files from PR and reverted `admin/SetLanguageFrequencies`.
Now there are only 3 file changes remaining:

- `po/languages.pot`
- `t/lib/t/MusicBrainz/Server/Controller/WS/js/Edit.pm`
- `t/sql/initial.sql`

Maybe the `.pot` file should not be changed either?

reosarevok commented:
Yes, the `.pot` file is also for translations and will be changed automatically.

jesus2099 commented:
I see I forgot a `po/languages_notrim.pot`.
I have reverted `po/languages.pot` now there are only 2 remaining changes:

- `t/lib/t/MusicBrainz/Server/Controller/WS/js/Edit.pm`
- `t/sql/initial.sql`

Are they OK?
If so, I have another PR where I must do some squash, so I will have to re-learn how to do that shortly and I will be able to make those 2 commits (more easy to follow-up now) into only 1.

jesus2099 commented:
I have changed 2 commits into 1.

reosarevok reviewed (approved):
This is good now I think.
